db/seeds.rb

```diff
@@ -13,3 +13,12 @@
 end
 
 FactoryBot.create_list(:billing_contract, 100)
+
+FactoryBot.create_list(:bill, 100)
+
+FactoryBot.create_list(:payment, 100)
+
+BillingContract.all.each do |billing_contract| 
+    FactoryBot.create(:bill, billing_contract: billing_contract)
+    FactoryBot.create(:payment, billing_contract: billing_contract)
```

Comment:
make some history for last 3 months per contract

so that there were 3 bills & payments for last 3 months
so that amount billed and payed is  equal
or has some difference so that contract will be over payed or under payed by a little